client-libraries/rust/src/reflector_config/mod.rs

```diff
@@ -1138,6 +1255,24 @@ override-timeline-attributes = ['c = true']
 protocol-child-port = 9079
 protocol-parent-url = 'modality-ingest://auxon.io:9077'
 
+[[ingest.rollover-trackers]]
```

Comment:
If I want to have two rollover trackers, it seems like I should be able to do that by adding another `[[ingest.rollover-trackers]]` at the bottom, right?

Reply:
yes, that's right